modules/primer-marketing-utilities/docs/padding.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Padding
+status: New release
+status_issue: https://github.com/github/design-systems/issues/378
+---
+
+Padding spacing utilities can be applied to elements to achieve bigger y-directional spaces beyond the `primer-core` spacing scale.
```

Comment:
Could you make the same changes to padding based on the changes I suggested for margins?